metadata/src/main/java/edu/unc/lib/dl/util/JMSMessageUtil.java

```diff
@@ -110,6 +110,7 @@ public static FedoraActions getAction(String value) {
         ADD("add"),
         EDIT_ACCESS_CONTROL("editAccess"),
         EDIT_TYPE("editType"),
+        ENHANCEMENTS("enhancements"),
```

Comment:
good idea to add a constant. My only suggestion is to make it a verb/action since all the other ones are